centreon/src/Core/AgentConfiguration/Domain/Model/ConfigurationParameters/TelegrafConfigurationParameters.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/*
+ * Copyright 2005 - 2023 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+declare(strict_types=1);
+
+namespace Core\AgentConfiguration\Domain\Model\ConfigurationParameters;
+
+use Centreon\Domain\Common\Assertion\Assertion;
+use Centreon\Domain\Common\Assertion\AssertionException;
+use Core\AgentConfiguration\Domain\Model\ConfigurationParametersInterface;
+
+/**
+ * @phpstan-type _TelegrafParameters array{
+ *	    otel_public_certificate: string,
+ *	    otel_ca_certificate: string|null,
+ *	    otel_private_key: string,
+ *	    conf_server_port: int,
+ *	    conf_certificate: string,
+ *	    conf_private_key: string
+ *  }
+ */
+class TelegrafConfigurationParameters implements ConfigurationParametersInterface
+{
+    public const BROKER_MODULE_DIRECTIVE = '/usr/lib64/centreon-engine/libopentelemetry.so /etc/centreon-engine/otl_server.json';
```

Comment:
/usr/lib64/centreon-engine could be dynamic as this value is defined in install step 3 https://docs.centreon.com/docs/installation/web-and-post-installation/#step-3-monitoring-engine-information